src/main/kotlin/domain/Participant.kt

```diff
@@ -13,5 +17,12 @@ abstract class Participant(
 
     fun displayCardInfo(): String = blackjackCards.displayCardInfo()
 
-    fun calculateScore(): Int = blackjackCards.calculateScore()
+    fun isBust(): Boolean = blackjackCards.calculateScore() > BlackjackCards.BLACKJACK_MAX_SCORE
+
+    fun isBlackjack(): Boolean =
+        blackjackCards.getCards().size == 2 && blackjackCards.calculateScore() == BlackjackCards.BLACKJACK_MAX_SCORE
+
+    fun applyProfit(profit: SignedMoney) {
+        this.profit = profit
+    }
 }
```

Comment:
```suggestion
fun applyProfit(profit: SignedMoney) = run { this.profit = profit }
```

참고로 이런 식으로 쓸 수도 있나봅니다 ㅎㅎ

Reply:
오호 저도 한번 찾아봤습니다 ㅎ (run 이라니.. 코틀린 어렵네요 ㅎㅎ)

처리 결과는 동일하고 이런 내용으로 검색되네용

```
run 을 사용한 표현식 형태도 가능하겠지만,
현재는 단순 필드 할당만 수행하고 있어서 Scope Function 을 사용할 이점은 없음.
기존 block 형태가 의도를 더 명확히 보여줌

scope function (run) 은 반환값을 활용하거나, 컨텍스트를 명확히 드러낼 때 의미가 있는데,
현재 코드에서는 단순 대입만 있어서 오히려 가독성을 해칠 수 있음
```

Reply:
넹넹 취향 문제인 것 같아서 미션에서는 편하신 것 선택해주시면 좋을 것 같아요 ㅎㅎ
근데 팀 레포에서는 컨벤션이 있으면 좋을 것 같네요! 같이 논의해봐요!